build.gradle

```diff
@@ -153,6 +155,39 @@ publishing {
                 }
             }
         }
+
+        shadow(MavenPublication) {
```

Comment:
We saw a signing issue with this approach do you not have to add this to the signing? Or is that being handled somehow

Reply:
Added signing dependencies, I verified the local publication has signatures now:

```
JackYe:0.1.2 yzhaoqin$ ls
s3-tables-catalog-for-iceberg-runtime-0.1.2-javadoc.jar		s3-tables-catalog-for-iceberg-runtime-0.1.2-sources.jar.asc
s3-tables-catalog-for-iceberg-runtime-0.1.2-javadoc.jar.asc	s3-tables-catalog-for-iceberg-runtime-0.1.2.module
s3-tables-catalog-for-iceberg-runtime-0.1.2-runtime.jar		s3-tables-catalog-for-iceberg-runtime-0.1.2.module.asc
s3-tables-catalog-for-iceberg-runtime-0.1.2-runtime.jar.asc	s3-tables-catalog-for-iceberg-runtime-0.1.2.pom
s3-tables-catalog-for-iceberg-runtime-0.1.2-sources.jar		s3-tables-catalog-for-iceberg-runtime-0.1.2.pom.asc
```